src/theme.js

```diff
@@ -1,33 +1,26 @@
 import { createMuiTheme } from '@material-ui/core/styles';
 
-import { colors } from './colors';
+import { colors } from 'colors';
 
 export const theme = createMuiTheme({
   palette: {
     primary: {
-      main: colors.violet,
+      main: colors.primaryColor,
     },
     secondary: {
-      main: colors.turquoise,
-      button: colors.buttonGray,
+      main: colors.secondaryColor,
+      button: colors.titleColor,
     },
   },
   typography: {
-    headline6: {
```

Comment:
I don't see `headline6` in `master` in `theme.js`: https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L34

How can this pull request delete from `master` something that is not present in `master`?
To be honest I don't understand how is that even possible :) Why is github showing this code as deleted.

Reply:
In this pull request you have commit for removing `headline6`:

![image](https://user-images.githubusercontent.com/6686679/115625228-6c790800-a304-11eb-974a-99eedcc0ecd0.png)

https://github.com/DarthVanger/NeuralNotes/pull/209/commits

But the same commit with the same number is in [master](https://github.com/DarthVanger/NeuralNotes/commits/master).

I don't understand why is github showing it as a new commit in this pull request.

On local when I do `git diff master` from this branch I don't see these changes shown by github.

Maybe try to create a new branch and a new pull request.

Or [this solution from stackoverflow](https://stackoverflow.com/a/46782679/1657101).